package.json

```diff
@@ -138,6 +138,7 @@
     "cheerio": "1.0.0-rc.3",
     "chokidar": "3.2.3",
     "cypress": "3.5.0",
+    "cypress-file-upload": "3.5.0",
```

Comment:
Note: Doing it in a "native way" is quite a nightmare, everyone was requesting a easier way. I was waiting for this package since I learned Cypress. Finally, it got released 2 weeks ago! [cypress-file-upload](https://www.npmjs.com/package/cypress-file-upload)

cc @taoeffect